packages/neuron-wallet/package.json

```diff
@@ -37,9 +38,11 @@
   },
   "dependencies": {
     "@nervosnetwork/ckb-sdk-core": "^0.0.1-alpha.3",
-    "electron-window-state": "^5.0.3",
     "@types/electron-store": "^1.3.0",
+    "ava": "^1.3.1",
     "electron-store": "^2.0.0",
+    "electron-window-state": "^5.0.3",
+    "execa": "^1.0.0",
```

Comment:
Is this NPM package for writing tests? If so perhaps it should go to `devDepencencies` instead?